exchanges/poloniex/poloniex_live_test.go

```diff
@@ -18,14 +21,46 @@ var mockTests = false
 func TestMain(m *testing.M) {
 	e = new(Exchange)
 	if err := testexch.Setup(e); err != nil {
-		log.Fatalf("Poloniex Setup error: %s", err)
+		log.Fatal(err)
 	}
+
 	if apiKey != "" && apiSecret != "" {
 		e.API.AuthenticatedSupport = true
+		e.API.AuthenticatedWebsocketSupport = true
 		e.SetCredentials(apiKey, apiSecret, "", "", "", "")
+		e.Websocket.SetCanUseAuthenticatedEndpoints(true)
 	}
-	log.Printf(sharedtestvalues.LiveTesting, e.Name)
+
 	e.Websocket.DataHandler = sharedtestvalues.GetWebsocketInterfaceChannelOverride()
 	e.Websocket.TrafficAlert = sharedtestvalues.GetWebsocketStructChannelOverride()
+	err := populateTradablePairs()
+	if err != nil {
+		log.Fatal(err)
```

Comment:
go test --tags=mock_test_off ./...
2025/10/03 09:34:37 futures asset type not enabled
FAIL    github.com/thrasher-corp/gocryptotrader/exchanges/poloniex      1.144s
FAIL